docs/reference/scylladb-limits.rst

```diff
@@ -0,0 +1,54 @@
+========================
+ScyllaDB Limits
+========================
+
+This document provides hard and soft limits in ScyllaDB.
+
+Cluster and Node Limits
+-----------------------------
+
+.. list-table:: 
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Component
+     - Limit 
+   * - Nodes per cluster
+     - Low hundreds
+   * - Node size
+     - 256 vcpu
+  
+See :ref:`Hardware Requirements <system-requirements-hardware>` for storage
+and memory requirements and limits.
+
+CQL Limits
+--------------
+
+.. list-table:: 
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Component
+     - Limit
+   * - Keyspaces per cluster 
+     - Thousands (tested with 1000)
+   * - Tables per keyspace 
+     - Low thousands (tested with 5000)
+   * - Tables per cluster (including indexes)
+     - Thousands
+   * - Columns per table
+     - Hundreds
+   * - Columns per cluster
+     - Tens of thousands
+   * - Views per table
+     - Low tens
+   * - Partition size
+     - Gigabytes
+   * - Rows per partition
+     - No limit
+   * - Row size
+     - Latency-related soft limit: 
+     
+       Hundreds of kilobytes (good latency) or megabytes (mediocre latency)
+   * - CDC
```

Comment:
Not clear what this is, is this the number of CDC-enabled tables?

Reply:
I sugget adding MV ans SI number per tables as well